docs/retail/loyalty-rewards/index.md

```diff
@@ -0,0 +1,321 @@
+---
+seo:
+  title: Loyalty Programmes
+  description: This recipe tracks customers' purchasing patterns, generating tailored rewards for a loyalty scheme.
+---
+
+# Customer Loyalty Schemes
```

Comment:
I've gotten feedback on other recipes to lead with an action verb, would you be able to do the same here?  `Building Customer Loyalty Schemas`...or the like

Reply:
`Crafting`? `Artisanal`? Is 'artisanal' a verb?

Let's just go with 'building'. :-D